README.md

```diff
@@ -27,18 +27,26 @@ can be found at nominatim.org as well.
 
 A quick summary of the necessary steps:
 
-1. Create a Python virtualenv and install the packages:
+
+1. Clone this git repository and download the country grid
+
+      git clone https://github.com/osm-search/Nominatim.git
+      wget -O Nominatim/data/country_osm_grid.sql.gz https://nominatim.org/data/country_grid.sql.gz
+
+2. Create a Python virtualenv and install the packages:
 
         python3 -m venv nominatim-venv
         ./nominatim-venv/bin/pip install packaging/nominatim-{api,db}
 
-2. Create a project directory, get OSM data and import:
+3. Create a project directory, get OSM data and import:
 
         mkdir nominatim-project
         cd nominatim-project
         ../nominatim-venv/bin/nominatim import --osm-file <your planet file>
+        <!-- Alternative command with logging: ../nominatim-venv/bin/nominatim import --osm-file <your planet file> 2>&1 | tee setup.log -->
```

Comment:
You can't use HTML syntax here. Also, this is getting too long for a quick setup. Lets just add the logging to the normal command and be done with it. I see that the full manual already does the same.